pkg/daemon/update.go

```diff
@@ -50,6 +50,38 @@ const (
 	realtimeKernelType = "realtime"
 )
 
+// HostInfo contains information of an OSTree based system
+type HostInfo struct {
```

Comment:
We already have `rpmOstreeState` in `rpm-ostree.go`; would be good to share code there.